### PR TITLE
Add dbt definitions

### PR DESCRIPTION
**Pull request by trlim**

### CODE REVIEW

**Comment by ncoish** [comment]
Is this PR being considered for merging in the near future?

File: src/um/dbt.rs [hunk 1]
```diff
@@ -0,0 +1,193 @@
...
+pub type PDEV_BROADCAST_NET = *mut DEV_BROADCAST_NET;
+STRUCT!{#[repr(packed)] struct DEV_BROADCAST_DEVICEINTERFACE_A {
+    dbcc_size: DWORD,
+    dbcc_devicetype: DWORD,
+    dbcc_reserved: DWORD,
+    dbcc_classguid: GUID,
+    dbcc_name: [c_char; 1], 
+}}
+pub type PDEV_BROADCAST_DEVICEINTERFACE_A = *mut DEV_BROADCAST_DEVICEINTERFACE_A;
+STRUCT!{#[repr(packed)] struct DEV_BROADCAST_DEVICEINTERFACE_W {
```
Comment (ncoish) [inline src/um/dbt.rs]: I ran in to difficulties while this struct was #[repr(packed)] on Windows 7 while testing this. RegisterDeviceNotificationA returned a null pointer, which fluctuated between getting a 0 and a 1066 from GetLastError(). The problem was resolved by removing #[repr(packed)].

Comment (trlim) [reply src/um/dbt.rs]: This is my mistake and I will remove them.
Thanks!

File: src/um/dbt.rs [hunk 2]
```diff
@@ -0,0 +1,193 @@
+// Copyright © 2016-2018 winapi-rs developers
```
Comment (retep998) [inline src/um/dbt.rs]: This should be just 2018.

**Review by retep998 — CHANGES_REQUESTED** [review]

**Review by retep998 — APPROVED** [review]

**Pull request merged**